.github/workflows/jest.yaml

```diff
@@ -1,23 +1,22 @@
 name: Jest
-on: [pull_request]
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+
 jobs:
-  Run-jest-tests:
+  call-pr-workflow:
+    uses: ./.github/workflows/pull_request.yml
+  
+  Jest:
     runs-on: ubuntu-latest
+    needs: call-pr-workflow
     steps:
-      - name: Check out repository code
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-
-      - name: Setup Node.js
-        uses: actions/setup-node@v1
-        with:
-          node-version: 18
-          registry-url: https://registry.npmjs.org
-
       - name: Jest
         run: |
+          pwd
```

Comment:
@reshav-abraham Getting this error when the job runs. Either the artifact isn't here, or doesn't include the package.json for some reason

```
npm ERR! enoent Could not read package.json: Error: ENOENT: no such file or directory, open '/home/runner/work/design_system/design_system/package.json'
```